packages/ui/src/theme/css/styles.scss

```diff
@@ -13,14 +13,15 @@
   }
 }
 
-[class*='amplify'] {
-  all: unset; /* protect against external styles */
+// Note: This rule can be easily overwritten when
+// needed due to very low specificity of 0 0 0
+* {
```

Comment:
Do we still need this? Taking a look at the original issue, https://github.com/aws-amplify/amplify-ui/issues/1092, it sounds like we don't want our styles seeping into other non-amplify ui components. Could this just be removed as all our components have classnames that start with 'amplify'?

Reply:
If we remove this, then we will be changing the default `box-sizing` behavior of the library for all of the children elements inside of `amplify` class components. For example, flex children which are not amplify components would have a different `box-sizing` behavior than amplify components.

  

**Without** `* { box-sizing: border-box; }`

```jsx

<Flex>

<div>1</div> {/* content-box */}

<Button>2</Button> {/* border-box */}

</Flex>

```

  

**With** `* { box-sizing: border-box; }`

```jsx

<Flex>

<div>1</div> {/* border-box */}

<Button>2</Button> {/* border-box */}

</Flex>

```

  

Leaving it off seems like it would lead to unexpected behavior for customers. What do you think @dbanksdesign